app/src/main/java/com/ivy/wallet/ui/loan/LoanViewModel.kt

```diff
@@ -55,8 +59,11 @@ class LoanViewModel @Inject constructor(
 
     private var defaultCurrencyCode = ""
 
+    private val _state = MutableStateFlow(LoanScreenState())
+    val state: StateFlow<LoanScreenState> = _state
+
     fun start() {
-        viewModelScope.launch {
+        viewModelScope.launch(Dispatchers.Default) {
```

Comment:
Why do we neec to specify `Disparchers.Default` on every launch?

Reply:
@ILIYANGERMANOV because by default the coroutine is launched in the main thread i.e the UI thread so to shift it to compute thread i did this 😅